mlflow_faculty/trackingstore.py

```diff
@@ -91,10 +98,18 @@ def get_experiment(self, experiment_id):
         :return: A single :py:class:`mlflow.entities.Experiment` object if it
             exists, otherwise raises an exception.
         """
-
-        faculty_experiment = self._client.get(self._project_id, experiment_id)
-
-        return faculty_experiment_to_mlflow_experiment(faculty_experiment)
+        try:
+            faculty_experiment = self._client.get(
+                self._project_id, experiment_id
+            )
+        except faculty.clients.base.HttpError as e:
+            raise MlflowException(
+                "{}. Received response {} with status code {}".format(
```

Comment:
I think we should be more specific here in translating HTTP like stuff like status codes into readable exception messages like `An experiment with that name already exists`.

Reply:
Currently the Error raised is 
`MlflowException: Name already used in project. Received response {"errorCode":"experiment_name_conflict","error":"Name already used in project"} with status code 409`

Are you suggesting to specifically match on the Conflict and write something even more explicit or to match on the `error_code` as we setting those?

Reply:
Oh, sorry, did not realize that `e.error` contained that information already.